benchmark_utils/optuna_solver.py

```diff
@@ -95,5 +97,9 @@ def get_result(self):
         # it is customizable for each benchmark.
         return self.clf
 
+    @staticmethod
+    def get_next(n_iter):

```

Comment:
You don't need to have a `staticmethod` anymore with the newest version of `benchopt`
```suggestion
    def get_next(self, n_iter):

```